docker-entrypoint.sh

```diff
@@ -131,6 +131,15 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		export RABBITMQ_CTL_ERL_ARGS="$RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS"
 	fi
 
+	echo
+	for f in /docker-entrypoint-initdb.d/*; do
+		case "$f" in
```

Comment:
@c0deaddict I think the sourcing can be useful, provided the underlying command uses environment variables directly.